yjit/src/codegen.rs

```diff
@@ -5807,15 +5806,15 @@ fn gen_send_general(
             // Can always call public methods
         }
         METHOD_VISI_PRIVATE => {
-            if flags & VM_CALL_FCALL == 0 {
+            if !flags.is_fcall() {
                 // Can only call private methods with FCALL callsites.
                 // (at the moment they are callsites without a receiver or an explicit `self` receiver)
                 return CantCompile;
             }
         }
         METHOD_VISI_PROTECTED => {
             // If the method call is an FCALL, it is always valid
-            if flags & VM_CALL_FCALL == 0 {
+            if flags.is_fcall() {
```

Comment:
```suggestion
            if !flags.is_fcall() {
```
was flipped by accident.

Reply:
Yeah, I literally did use that comment when I made the code change. Changed it to be clearer imo

Thanks for the catch.